SPECS/libiscsi/libiscsi.spec

```diff
@@ -1,98 +1,71 @@
-#######################################################################
-
-# Conflict with iscsi-initiator-utils.
-
-%global libiscsi_includedir %{_includedir}/iscsi
-%global libiscsi_libdir %{_libdir}/iscsi
-
-Summary:        iSCSI client library
-Name:           libiscsi
-Version:        1.18.0
-Release:        11%{?dist}
-License:        GPLv2+ AND LGPLv2+
-Vendor:         Microsoft Corporation
-Distribution:   Mariner
-URL:            https://github.com/sahlberg/%{name}
-Source:         https://github.com/sahlberg/%{name}/archive/%{version}.tar.gz#/%{name}-%{version}.tar.gz
-# Fix gcc7 warnings
-Patch0001:      0001-Fix-compiler-warning-error-under-gcc7.patch
-Patch0002:      0002-Fix-another-compiler-warning.patch
-# Fix 32bit build
-Patch0003:      0003-Fix-32bit-build.patch
-# Fix build with latest rdma-core
-# Submitted upstream: https://github.com/sahlberg/libiscsi/pull/265
-Patch0004:      0004-iser-Use-local-container_of-definition.patch
-Patch0005:      0005-fix-ipv6.patch
-Patch0006:      0006-fix-coverity-report.patch
-BuildRequires:  CUnit-devel
-BuildRequires:  autoconf
-BuildRequires:  automake
-BuildRequires:  libgcrypt-devel
-BuildRequires:  libtool
-BuildRequires:  popt-devel
+Name: libiscsi
+Summary: iSCSI client library
+Version: 1.19.0
+Release: 1%{?dist}
+License: LGPLv2+
+URL: https://github.com/sahlberg/%{name}
+
+Source: https://github.com/sahlberg/%{name}/archive/%{version}.tar.gz#/%{name}-%{version}.tar.gz
+
+BuildRequires: make
+BuildRequires: autoconf
+BuildRequires: automake
+BuildRequires: libtool
+BuildRequires: popt-devel
+BuildRequires: CUnit-devel
+BuildRequires: libgcrypt-devel
 %ifnarch %{arm}
-BuildRequires:  rdma-core-devel
+BuildRequires: rdma-core-devel
 %endif
 
 %description
 libiscsi is a library for attaching to iSCSI resources across
 a network.
 
-%package devel
-Summary:        iSCSI client development libraries
-License:        GPLv2+ AND LGPLv2+
-Requires:       %{name} = %{version}-%{release}
 
-%description devel
-The libiscsi-devel package includes the header files for libiscsi.
+#######################################################################
 
-%package utils
-Summary:        iSCSI Client Utilities
-License:        GPLv2+
-Requires:       %{name} = %{version}-%{release}
+# Conflict with iscsi-initiator-utils.
```

Comment:
What does this comment mean? Any additional info that can be added?

Reply:
i don't have any knowledge here ... the comment has always been there, it stands out because this PR seems to have moved it.